app/views/steps/check/results/shared/_disclaimer.en.html.erb

```diff
@@ -19,3 +19,10 @@
   If you're not sure if you should tell people about a caution or conviction,
   you should consider getting specific legal advice. This checker offers general guidance only.
 </p>
+
+<p class="govuk-body">
+  If you need to change any information, or if you have another caution or conviction you’d like to check,
+  start a new check.
+</p>
+
+<%= link_to 'Start a new check', root_path, class: 'govuk-button govuk-button--start govuk-!-margin-top-2', data: { ga_category: 'results', ga_label: 'new check' } %>.
```

Comment:
You can use a helper to create these "buttonized"  links.
Take a look at the file `app/views/warning/reset_session.html.erb` for example, as it also uses GA tracking 👍

Reply:
Oh nice one! Have changed this. Thanks for that 😃